.github/workflows/release.yml

```diff
@@ -15,8 +15,9 @@ permissions:
   contents: read
 
 env:
-  ELASTIC_CDN_BUCKET_NAME: ${{ inputs.dry-run == false && 'apm-rum-357700bc' || 'oblt-apm-agent-rum-js-ci' }}
-  ELASTIC_CDN_CREDENTIALS: ${{ inputs.dry-run == false && 'secret/gce/elastic-cdn/service-account/apm-rum-admin' || 'secret/observability-team/ci/service-account/apm-agent-rum-js' }}
+  ELASTIC_CDN_BUCKET_NAME: ${{ inputs.dry-run == false && 'apm-rum-357700bc' || 'oblt-apm-rum-test' }}
+  ELASTIC_CDN_PROJECT_ID: ${{ inputs.dry-run == false && 'elastic-cdn-4ae000ab' || 'elastic-observability-ci' }}
+  ELASTIC_CDN_PROJECT_NUMBER: ${{ inputs.dry-run == false && '382950469386' || '911195782929' }}
```

Comment:
There is no way to use one or the other but both?

Reply:
Thanks for the hint.

Seems like in existing cases we are only using the project-number.

Reply:
https://github.com/elastic/apm-agent-rum-js/pull/1540/commits/4f8a3f569e1baeab5eea3cb164bc83032c8bbbf4

The dry-run of that commit was successful here: https://github.com/elastic/apm-agent-rum-js/actions/runs/10879975420